### PR TITLE
Update gallery section of README.md and demos/README.md

### PR DESCRIPTION
**Pull request by xhlulu**
## About
 

Added a bunch of images to the gallery of section of both readmes.

## Pre-Merge checklist
- [x] The project was correctly built with `npm run build:all`.
- [x] If there was any conflict, it was solved correctly
- [x] All changes were documented in CHANGELOG.md.
- [x] All tests on CircleCI have passed.
- [x] All Percy visual changes have been approved.
- [ ] Two people have :dancer:'d the pull request. You can be one of these people if you are a Dash Cytoscape core contributor.

## Reference Issues

Closes #[issue number]

## Other comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,8 +103,26 @@ The Pull Request and Issue Templates were inspired from the
...
+
+### Create your own stylesheet
+[Code](usage-advanced.py) | [Demo](https://dash-gallery.plotly.host/cytoscape-advanced)
+![View usage-advanced on Github](demos/images/usage-advanced.gif)
+
+### Using event callbacks
+[Code](usage-events.py) | [Demo](https://dash-gallery.plotly.host/cytoscape-events)
+![View usage-events on Github](demos/images/usage-events.gif)
+
+### Using external layouts
```
Comment (mkcor) [inline README.md]:
```suggestion
### Use external layouts
```

File: README.md [hunk 2]
```diff
@@ -103,8 +103,26 @@ The Pull Request and Issue Templates were inspired from the
...
+
+### Automatically generate interactive phylogeny trees
+[Code](demos/usage-phylogeny.py) | [Demo](https://dash-gallery.plotly.host/cytoscape-phylogeny/)
+![View usage-phylogeny on Github](demos/images/usage-phylogeny.gif)
+
+### Create your own stylesheet
+[Code](usage-advanced.py) | [Demo](https://dash-gallery.plotly.host/cytoscape-advanced)
+![View usage-advanced on Github](demos/images/usage-advanced.gif)
+
+### Using event callbacks
```
Comment (mkcor) [inline README.md]:
```suggestion
### Use event callbacks
```

**Comment by mkcor** [comment]
@xhlulu could you please change the `.PNG` file extensions into `.png`? Thanks!

**Comment by xhlulu** [comment]
I've never used that before, thanks for sharing 😆

**Comment by mkcor** [comment]
> I've never used that before, thanks for sharing :laughing:

You're welcome @xhlulu :)

**Pull request merged**